pkg/service/storage/ucan/replica_allocate.go

```diff
@@ -128,7 +128,7 @@ func ReplicaAllocate(storageService ReplicaAllocateService) server.Option {
 				if err := storageService.Replicator().Replicate(ctx, &replicahandler.TransferRequest{
 					Space:  cap.Nb().Space,
 					Blob:   cap.Nb().Blob,
-					Source: replicaAddress,
+					Source: replicahandler.TransferSource{ID: claim.Issuer(), URL: replicaAddress},
```

Comment:
Confirming my understanding: Issuer of the location claim is the node that must have published the content.